<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 72, 102);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>691</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(51, 51, 51);
border-radius: 15px
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="name">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>461</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;

}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>                                Fullname</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="course">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>240</y>
         <width>461</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;

}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>                             Course &amp; Year</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ok">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>410</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(200, 0, 0);
color: rgb(255, 255, 255);
border-radius:15px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#49ebff;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_preview">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>410</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(200, 0, 0);
color: rgb(255, 255, 255);
border-radius:15px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#49ebff;
}</string>
       </property>
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maximumTime">
        <time>
         <hour>18</hour>
         <minute>30</minute>
         <second>59</second>
        </time>
       </property>
       <property name="minimumTime">
        <time>
         <hour>7</hour>
         <minute>30</minute>
         <second>0</second>
        </time>
       </property>
       <property name="time">
        <time>
         <hour>7</hour>
         <minute>30</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
      <widget class="QComboBox" name="room">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>320</y>
         <width>69</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>----</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
